src/main/java/duke/task/AddTask.java

```diff
@@ -0,0 +1,128 @@
+package duke.task;
+
+import duke.Duke;
+import duke.storage.Storage;
+import duke.design.Default;
+
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class AddTask {
+    /**
+     * Performs the add task action
+     *
+     * @param tasks   The array list that stores all the tasks
+     * @param command The input command typed by the user
+     * @param words   The array of words that compose the input command
+     */
+    public static void addTask(ArrayList<Task> tasks, String command, String[] words) throws IOException {
+        if (isCorrectToDo(tasks, command, words) || isCorrectDeadline(tasks, command, words) || isCorrectEvent(tasks, command, words)) {
+            Default.showMessage(" Class type [" + tasks.get(tasks.size() - 1).getClassType() + "] \"" + tasks.get(tasks.size() - 1) +
```

Comment:
Consider building the string first to [avoid complicated expressions](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-complicated-expressions).